testing/testcafe/tests/scheduler/grouping/smoothCellLines.ts

```diff
@@ -0,0 +1,40 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { changeTheme } from '../../../helpers/changeTheme';
+import createWidget from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+import Scheduler from '../../../model/scheduler';
+
+fixture.disablePageReloads`Scheduler: SmoothCellLines`
+  .page(url(__dirname, '../../container.html'));
+
+const resourcesData = [...Array(20).keys()].map((num: number) => ({
+  text: `Name ${num}`,
+  id: num,
+}));
+
+test('The group panel and date table stay in sync during scrolling on material themes (T1146448)', async (t) => {
+  const scheduler = new Scheduler('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await t.scroll(scheduler.element.find('.dx-scrollable-container').nth(1), 0, 1100);
```

Comment:
Let's use the existing ```workspaceScrollable``` from the Scheduler PageObjecModel instead of the raw CSS selector:

```suggestion
  await t.scroll(scheduler.workspaceScrollable, 0, 1100);
```